tests/test_Step.cc

```diff
@@ -358,8 +358,8 @@ TEST_CASE("execute(): C++ exceptions", "[Step]")
     {
         step.set_script("throw_logic_error(); a = 42");
 
-        // A C++ exception bubbles through the Lua execution callstack, but must be caught
-        // by Sol2, returned as a protected_function_result, and reported as a task::Error.
+        // A C++ exception must be intercepted by Sol2's call trampoline and finally
```

Comment:
Must that not be called 'Sol3', as they already talk about `Sol4` in the commit messages / release page.

Reply:
I can never tell with Sol2/3. But will go with Sol3 from now on.